docs/source/federated-types/composition.mdx

```diff
@@ -85,16 +85,16 @@ type Event {
 
 One subgraph defines `Event.timestamp` as a `String`, and the other defines it as an `Int`. Composition doesn't know which type to use, so it fails.
 
-> For examples of _valid_ inconsistencies in field return types, see [Varying field return types](./value-types/#varying-field-return-types).
+> For examples of _valid_ inconsistencies in field return types, see [Varying shared field return types](./sharing-types/#varying-shared-fields).
 
 Breaking composition is a helpful feature of federation! Whenever a team modifies their subgraph schema, those changes might conflict with _another_ subgraph. But that conflict won't affect your gateway, because composition fails to generate a new supergraph schema. It's like a compiler error that prevents you from running invalid code.
 
 ## Rules of composition
-
 In Federation 2, your subgraph schemas must follow **all** of these rules to successfully compose into a supergraph schema:
 
-* If multiple subgraphs define the same field, that field must have a compatible return type _and_ compatible argument types across each defining subgraph.
-    * For examples of compatible and incompatible field return types, see [Varying field return types](./value-types/#varying-field-return-types).
+* Multiple subgraphs can not [redefine the same field on an Object type](./sharing-types/#explicit-and-implicit-type-sharing) _unless_ that field is shared.
```

Comment:
"can not" -> "can't"

"redefine" -> "define"

"Object" -> "object"

Comma before "unless"

I'd recommend moving the link to just be the word "shared" at the end of this sentence, because "sharing" is the core subject of the target